ConsentViewController/Classes/SPConsentManager.swift

```diff
@@ -46,20 +46,22 @@ import Foundation
         }
     }
 
-    public convenience init(accountId: Int, propertyName: SPPropertyName, campaigns: SPCampaigns, delegate: SPDelegate?) {
+    public convenience init(accountId: Int, propertyName: SPPropertyName, campaignEnv: SPCampaignEnv = .Public, campaigns: SPCampaigns, delegate: SPDelegate?) {
```

Comment:
@vilasma let's rename the parameter from `campaignEnv` to `campaignsEnv`. I think this makes it a bit more explicit this parameter refers to all campaigns.

Reply:
Done